### PR TITLE
ORSR app for missing stakeholders identifiers: Parse stakeholder prefixes from RPO formatted name if missing in structured data

### PR DESCRIPTION
**Pull request by luciajanikova**

### CODE REVIEW

File: app/lib/upvs_submissions/forms/fuzs_data.rb [hunk 1]
```diff
@@ -248,6 +249,16 @@ def with_missing_municipality_identifier?
 
         private
 
+        def parse_person_formatted_name(formatted_name)
+          return if @person_prefixes.present? || @person_postfixes.present?
+
+          formatted_name_pattern = /(.*)\b#{given_name}\b\s\b#{family_name}\b(.*)/
+          prefixes, postfixes = formatted_name.match(formatted_name_pattern).captures
```
Comment (jsuchal) [inline app/lib/upvs_submissions/forms/fuzs_data.rb:256]: Tu nestaci `split`?

**Review by jsuchal — APPROVED** [review]
Pozri este ten split a inak dobre

File: app/lib/upvs_submissions/forms/fuzs_data.rb (hunk 1 above)
Comment (luciajanikova) [reply app/lib/upvs_submissions/forms/fuzs_data.rb:256]: Split ako, podla coho @jsuchal?

Comment (jsuchal) [reply app/lib/upvs_submissions/forms/fuzs_data.rb:258]: Aha, ono to asi funguje rovnako. Ja som myslel nieco taketo `prefix, suffix = s.split(/Jan Suchal/, 2)`

**Pull request merged**